src/screens/AccountImportMethod/AccountImportMethod.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ */
+
+import React, { PureComponent } from 'react'
+import {
+  Image,
+  Text,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+import textConstants from '../../locales/en'
+import PropTypes from 'prop-types'
+import TextButton from '../../components/TextButton'
+import styles from './AccountImportMethodStyles'
+
+export default class SelectAccountImportMethod extends PureComponent {
+  renderMethod = (accountImportMethod) => (
+    <Method
+      key={accountImportMethod.id}
+      {...accountImportMethod}
+      onPress={this.props.onSelectAccountImportMethod(accountImportMethod)}
+    />
+  )
+
+  render() {
+    const {
+      accountImportMethods,
+      onCreateWallet,
+    } = this.props
+
+    return (
+      <View>
+        <View style={styles.buttons}>
+          {accountImportMethods.map(this.renderMethod)}
+        </View>
+
+        <Text style={styles.or}>
+          {textConstants.or}
+        </Text>
+        <TextButton
+          label='Create new wallet'
+          onPress={onCreateWallet}
+        />
+      </View>
+    )
+  }
+}
+
+SelectAccountImportMethod.propTypes = {
+  accountImportMethods: PropTypes.arrayOf(
+    PropTypes.shape({
+      id: PropTypes.string,
+      label: PropTypes.string,
+      screen: PropTypes.string,
+      title: PropTypes.string,
+    })
+  ),
+  onCreateWallet: PropTypes.func,
+  onSelectAccountImportMethod: PropTypes.func,
+}
+
+class Method extends PureComponent {
```

Comment:
I suggest to move this class into `src/screens/AccountImportMethod/Method.js` (yes, right here, because it would not reused in other parts of code).